app/settings_strings.grdp

```diff
@@ -51,6 +51,24 @@
     <message name="IDS_SETTINGS_ABOUT_PAGE_SEND_FEEDBACK" desc="Text of the button which allows the user to send feedback with Brave.">
       Send feedback
     </message>
+    <message name="IDS_SETTINGS_GET_THE_MOST_OUT_OF_CHROME" desc="Section title for the 'Get the most out of Brave' page.">
+      Get the most ouf of Brave
```

Comment:
```suggestion
      Get the most out of Brave
```

Reply:
Good catch, although that error is in the upstream string and it's only relevant `<if expr="_google_chrome">` so I think we can safely ignore it. cc: @mkarolin just in case I'm wrong about that!